core/templates/core/navbar.html

```diff
@@ -0,0 +1,23 @@
+<nav class="flex flex-nowrap justify-between w-full p-1 bg-gray-950 border-b border-gray-600 text-gray-300">
```

Comment:
Why not in `templatetahs`?

Reply:
em I did it the same way with `list_events.html` and I thought that you should put template which is assigned to templatetag to templates